v3/plugins/sonarsource/sonarlint-vscode/0.0.1/meta.yaml

```diff
@@ -12,6 +12,6 @@ category: Linter
 repository: https://www.sonarlint.org/
 spec:
   containers:
-    - image: "garagatyi/remotetheia:java"
+    - image: "eclipse/che-remote-plugin-runner-java11:next"
```

Comment:
I thought sonarlint was not working with new versions of java. It has been fixed?

Reply:
looks like you are right, in the documentation I see that requirement is Java Runtime (JRE) 8.
So, I've changed it back to use java 8. Thanks!